app/contexts/movies/commands/update.rb

```diff
@@ -0,0 +1,18 @@
+module Contexts
+  module Movies
+    module Commands
+      class Update
+        attr_reader :repository
+
+        def initialize(repository = Movie)
+          @repository = repository
+        end
+
+        def call(params)
+          Movie.find(params[:id])
+          repository.update(params)
```

Comment:
not sure if it updates any records, 
`repository` is a class(Movie) so it's not instance of Movie, so `update` won't work on it 

also why do you need `Movie.find(params[:id])`? you just getting record and then do nothing 
it would make sense to do smth like this: 
```ruby 
movie = repository.find(params[:id])
movie.update(params)
```

```suggestion
          repository.find(params[:id]).update(params)
```
let's be consistent by using only Model itself or repository